### PR TITLE
Install SC after upgrading from 3.6 to 3.7

### PR DESCRIPTION
**Pull request by sdodson**
Surely there's a better way?

### CODE REVIEW

**Comment by sdodson** [comment]
FWIW, I haven't personally run this playbook to know if it even works post install or not so please don't merge this just yet.

File: sjb/config/test_cases/test_branch_origin_extended_conformance_install_update.yml [hunk 1]
```diff
@@ -169,6 +169,30 @@ extensions:
...
+                            --become               \
+                            --become-user root     \
+                            --connection local     \
+                            --inventory sjb/inventory/ \
+                             "${sc_playbook}"     \
+                            -e etcd_data_dir="${ETCD_DATA_DIR}" \
+                            -e openshift_pkg_version="-${ORIGIN_UPGRADE_RELEASE_VERSION}" \
+                            -e deployment_type=$( cat ./DEPLOYMENT_TYPE)                  \
+                            -e openshift_node_port_range='30000-32000'                    \
+                            -e 'osm_controller_args={"enable-hostpath-provisioner":["true"]}' \
```
Comment (stevekuznetsov) [inline sjb/config/test_cases/test_branch_origin_extended_conformance_install_update.yml]: some of these look spurious?

**Comment by staebler** [comment]
@sdodson I have run that playbook post-install. In fact, most of my testing for my recent changes was done by running that playbook post-install.

File: sjb/config/test_cases/test_branch_origin_extended_conformance_install_update.yml (hunk 1 above)
Comment (sdodson) [reply sjb/config/test_cases/test_branch_origin_extended_conformance_install_update.yml]: My main intent was just to ensure we used the same exact same vars as previous playbooks within this job.

File: sjb/config/test_cases/test_branch_origin_extended_conformance_install_update.yml [hunk 2]
```diff
@@ -169,6 +169,30 @@ extensions:
                           -e 'osm_controller_args={"enable-hostpath-provisioner":["true"]}' \
                           -e oreg_url='openshift/origin-${component}:'"$( cat ./ORIGIN_COMMIT )"
     - type: "script"
+      title: "Install service catalog"
+      repository: "aos-cd-jobs"
+      timeout: 1800
+      script: |-
+        if [[ "${ORIGIN_UPGRADE_RELEASE_VERSION}" > 3.6 ]]; then
```
Comment (jim-minter) [inline sjb/config/test_cases/test_branch_origin_extended_conformance_install_update.yml]: better remove this line before we reach 3.10 ... :smile:

Comment (sdodson) [reply sjb/config/test_cases/test_branch_origin_extended_conformance_install_update.yml]: thanks, updated to use ORIGIN_UPGRADE_RELEASE_MINOR_VERSION instead

**Pull request closed without merging**